core/divelist.c

```diff
@@ -823,7 +819,7 @@ void process_loaded_dives()
 
 	fulltext_populate();
 
-	/* Inform frontend of reset data. This should reset all the models. */
+	/* Inform frontend of reset data. This should reset all the models and the filter. */
 	emit_reset_signal();
```

Comment:
is there something implicit that I'm missing - I don't see a `connect` call of that new `reset()` slot to the signal...

Reply:
Good catch - this comment change is an artifact of a previous version. Removed.